src/util/pg.py

```diff
@@ -1621,3 +1622,86 @@ def create_id_sequence(cr, table, set_as_default=True):
                 table=table_sql,
             )
         )
+
+
+def update_table_from_dict(cr, table, columns, mapping, key_col="id", bucket_size=DEFAULT_BUCKET_SIZE):
```

Comment:
Same here. We could remove the bucket_size parameter. In most cases when we **need** to use a dict update is because some process was necessarily done in Python. Thus it should be the caller responsibility to chunk it anyway. Moreover this would allow using the new util from a callback used in a process pool for example.